src/main/java/org/zone/region/flag/entity/monster/move/MonsterPreventionListener.java

```diff
@@ -36,4 +37,23 @@ public void onMonsterMoveForMonsterPrevention(MoveEntityEvent event,
         }
         event.setCancelled(true);
     }
-}
+
+    @Listener
+    public void onMonsterSpawnEvent(SpawnEntityEvent event, Monster monster) {
```

Comment:
Remove the ``Monster monster``

Events can only take 1 parameter (that being the event) if you specify more parameters, you need a annotation

Reply:
ok...